src/EnergyPlus/SimAirServingZones.hh

```diff
@@ -237,7 +237,6 @@ struct SimAirServingZonesData : BaseGlobalStruct
     Real64 Vot = 0.0;            // Required outdoor air intake at primary AHU per ASHRAE std 62.1
 
     int TUInNode = 0;            // inlet node number of a terminal unit
-    Real64 SumZoneDesFlow = 0.0; // sum of the zone design air mass flow rates for zones served by a system
     Real64 OAReliefDiff = 0.0;   // local for massflow change across OA system, kg/s
```

Comment:
Removed, this variable was used to calculate variable `SysToZoneDesFlowRatio`, which is no longer needed.